.taskcluster.yml

```diff
@@ -78,7 +78,6 @@ tasks:
           retries: 5
           routes:
             - statuses  # Automatically added by taskcluster-github. It must be explicit because of Chain of Trust
-            - tc-treeherder.v2.fenix.${head_rev}
```

Comment:
Nit: Let's make this route conditional. I had started https://github.com/mozilla-mobile/fenix/pull/2921. I'm fine closing it so it's merged in this PR.

Reply:
Let's keep your PR, my route-removal was temporary